atomic_reactor/util.py

```diff
@@ -640,14 +642,18 @@ def get_credentials(self, docker_registry):
             return {}
 
 
-class ManifestDigest(object):
+class ManifestDigest(dict):
     """Wrapper for digests for a docker manifest."""
 
-    def __init__(self, v1=None, v2=None, v2_list=None, oci=None):
-        self.v1 = v1
-        self.v2 = v2
-        self.v2_list = v2_list
-        self.oci = oci
+    content_type = {
+        'v1': MEDIA_TYPE_DOCKER_V2_SCHEMA1,
+        'v2': MEDIA_TYPE_DOCKER_V2_SCHEMA2,
+        'v2_list': MEDIA_TYPE_DOCKER_V2_MANIFEST_LIST,
+        'oci': MEDIA_TYPE_OCI_V1
+    }
+
+    def __init__(self, **kwargs):
+        self.update(**kwargs)
```

Comment:
Think this is how `dict` already behaves. Maybe we don't need `__init__` here?

Reply:
Agreed, fixed